README.md

```diff
@@ -267,6 +267,12 @@ _The embedding is confirmed to work as expected. It returns the right document,
 
 **The dry run will still consume tokens to embed your query, but it is only ~1/15 of the prompt.**
 
+## Colab Notebook and Video Tutorials
+
+Chinese Colab Tutorial:https://colab.research.google.com/drive/10_7Y0x4YXWVjuhhYwVraGQLpKAatTQTm?usp=sharing
+
+Chinese Video Tutorial:https://www.bilibili.com/video/BV1YX4y1H7oN
```

Comment:
Hey, one small request.

can you please add your video tutorial in this repo? https://github.com/embedchain/community-showcase

Idea is to have a community showcase repo where all the tutorials can be clubbed together.

Reply:
sure